CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/attribute/AbstractAttributeDescription.java

```diff
@@ -53,108 +53,93 @@ public NativeAttributeType getNativeType() {
     }
 
     @Override
-    public void setShort(int id, short value) {
+    public void setShort(final int id, final short value) {
         throw new IllegalArgumentException(String.format("Error converting short to %s", getName()));
     }
 
     @Override
-    public int getInt(int id) {
+    public int getInt(final int id) {
         throw new IllegalArgumentException(String.format("Error converting %s to int", getName()));
     }
 
     @Override
-    public void setInt(int id, int value) {
+    public void setInt(final int id, final int value) {
         throw new IllegalArgumentException(String.format("Error converting int to %s", getName()));
     }
 
     @Override
-    public long getLong(int id) {
+    public long getLong(final int id) {
         throw new IllegalArgumentException(String.format("Error converting %s to long", getName()));
     }
 
     @Override
-    public void setLong(int id, long value) {
+    public void setLong(final int id, final long value) {
         throw new IllegalArgumentException(String.format("Error converting long to %s", getName()));
     }
 
     @Override
-    public float getFloat(int id) {
+    public float getFloat(final int id) {
         throw new IllegalArgumentException(String.format("Error converting %s to float", getName()));
     }
 
     @Override
-    public void setFloat(int id, float value) {
+    public void setFloat(final int id, final float value) {
         throw new IllegalArgumentException(String.format("Error converting float to %s", getName()));
     }
 
     @Override
-    public double getDouble(int id) {
+    public double getDouble(final int id) {
         throw new IllegalArgumentException(String.format("Error converting %s to double", getName()));
     }
 
     @Override
-    public void setDouble(int id, double value) {
+    public void setDouble(final int id, final double value) {
         throw new IllegalArgumentException(String.format("Error converting double to %s", getName()));
     }
 
     @Override
-    public boolean getBoolean(int id) {
+    public boolean getBoolean(final int id) {
         throw new IllegalArgumentException(String.format("Error converting %s to boolean", getName()));
     }
 
     @Override
-    public void setBoolean(int id, boolean value) {
+    public void setBoolean(final int id, final boolean value) {
         throw new IllegalArgumentException(String.format("Error converting boolean to %s", getName()));
     }
 
     @Override
-    public char getChar(int id) {
+    public char getChar(final int id) {
         throw new IllegalArgumentException(String.format("Error converting %s to char", getName()));
     }
 
     @Override
-    public void setChar(int id, char value) {
+    public void setChar(final int id, final char value) {
         throw new IllegalArgumentException(String.format("Error converting char to %s", getName()));
     }
 
     @Override
-    public String getSearchString(int id) {
```

Comment:
From what I can tell this is removed because it hasn't been used anywhere by any other class.

Reply:
Correct. I think there were a couple of methods I removed from `AttributeDescription` for the same reason.